appservice/src/SiteWrapper.ts

```diff
@@ -225,6 +234,20 @@ export class SiteWrapper {
         return await this.waitForDeploymentToComplete(kuduClient, outputChannel);
     }
 
+    public async showScmPrompt(currentScmType: string): Promise<string> {
+        const placeHolder: string = localize('scmPrompt', 'Current ScmType is "{0}".  Select a new deployment source.', currentScmType);
+        const scmQuickPicks: vscode.QuickPickItem[] = [];
+        // generate quickPicks to not include current type
+        for (const scmQuickPick of SCM_TYPES) {
```

Comment:
Rather than hiding the current active scmType, should I include all four types but just change the description/detail to indicate that it is the current active scmType?